server/lib/chat/chat-service.ts

```diff
@@ -253,23 +334,23 @@ export default class ChatService {
       )
     }
 
-    const originalChannelName = await this.getOriginalChannelName(channelName)
-    if (originalChannelName === 'ShieldBattery') {
+    if (channelId === 1 && !CAN_LEAVE_SHIELDBATTERY_CHANNEL) {
+      // TODO(2Pac): Remove this restriction once we have a news/home page
```

Comment:
Just put this todo around the feature flag and not everywhere that you reference the feature flag